### ISSUE TITLE
Underscore in password regex missing

### ISSUE THREAD

**Issue by KingXP-Pythoner** [SHIV5T3R/CO-DE#60]
I'm not sure if it was intentional or not but it is not possible to create an account using a password with underscores. I believe new user should be allowed to do this.

**Comment by MrBisquit** [comment]
I'm not sure I've ever seen a password use underscores in it.

**Comment by KingXP-Pythoner** [comment]
It makes no sense to me to prohibit it. I don't think there's even any popular security convention that says to prohibit it. Users should be able to formulate their passwords however they want as long as the basic guards are set in place like minimum of 8 characters and so on

**Comment by MrBisquit** [comment]
> It makes no sense to me to prohibit it. I don't think there's even any popular security convention that says to prohibit it. Users should be able to formulate their passwords however they want as long as the basic guards are set in place like minimum of 8 characters and so on

Yeah, but I'm not sure anywhere let's you use an underscore in a password.

**Comment by KingXP-Pythoner** [comment]
> > It makes no sense to me to prohibit it. I don't think there's even any popular security convention that says to prohibit it. Users should be able to formulate their passwords however they want as long as the basic guards are set in place like minimum of 8 characters and so on
> 
> Yeah, but I'm not sure anywhere let's you use an underscore in a password.

I'm sorry what? Literally all my life I've never encountered restriction of underscores. This is becoming an unnecessary point of contention to be honest

**Comment by zatoa** [comment]
Actually I rarely use the letter 'b' in my passwords. Mind if we block those too?

**Comment by MrBisquit** [comment]
I mean, if you want to, I don't see why not.

**Comment by KingXP-Pythoner** [comment]
> I mean, if you want to, I don't see why not.

So why were you bringing up some sort of wall against it? That was odd mate.

**Comment by MrBisquit** [comment]
> > I mean, if you want to, I don't see why not.
> 
> So why were you bringing up some sort of wall against it? That was odd mate.

I'm not, I'm just saying that I've never seen a password with an underscore in it, that's all.

**Comment by zatoa** [comment]
You can't just edit the comments to suggest something else and pretend what @KingXP-Pythoner was mentioning doesn't exist. Just own up to it and say "Yeah you're right my bad" so when someone else views this thread they don't think @KingXP-Pythoner is making a fuss over nothing.

**Comment by KingXP-Pythoner** [comment]
> You can't just edit the comments to suggest something else and pretend what @KingXP-Pythoner was mentioning doesn't exist. Just own up to it and say "Yeah you're right my bad" so when someone else views this thread they don't think @KingXP-Pythoner is making a fuss over nothing.

Woah, I just noticed the edit 😕👎
